libcontainerd/container_linux.go

```diff
@@ -194,3 +195,16 @@ func (ctr *container) handleEvent(e *containerd.Event) error {
 	}
 	return nil
 }
+
+func (ctr *container) discardFifos() {
+	for _, i := range []int{syscall.Stdout, syscall.Stderr} {
+		f := ctr.fifo(i)
+		c := make(chan struct{})
+		go func() {
+			close(c)
```

Comment:
I remember you described point of this trick before, but would you mind to add comment or everybody will wanna remove it :)